### PR TITLE
Update UG with more descriptions

### PR DESCRIPTION
**Pull request by luo-git**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -124,7 +142,9 @@ Examples:
 
 ### Adding a label to a tag : `label`
 
-Adds one or more label to an existing tag. Duplicated labels will only be added once. 
+Adds one or more label to an existing tag.<br>
+You can use label to categorise tags.
+Warning: Duplicated labels will only be added once.
```
Comment (HynRidge) [inline docs/UserGuide.md:152]: Can the Warning be in the information source thingy? (sorry i don't know what is the component called, but it was something like a box of)

**Review by HynRidge — APPROVED** [review]
Insane effort Mr.Grammar!!
LGTM!!!
I just have some minor issues regarding one of the format

File: docs/UserGuide.md (hunk 1 above)
Comment (luo-git) [reply docs/UserGuide.md:152]: Done!

**Review by HynRidge — APPROVED** [review]
👍 LGTM!!

**Pull request merged**